packages/dashboard/src/components/dialog/stories/index.js

```diff
@@ -35,18 +33,30 @@ import Dialog from '../dialog';
 export default {
   title: 'Dashboard/Components/Dialog/Base',
   component: Dialog,
+  args: {
+    isOpen: true,
+    title: 'my dialog title',
+    secondaryText: 'cancel action',
+    primaryText: 'confirm action',
+  },
+  argTypes: {
+    onClose: {
+      action: 'closed',
+    },
+    onPrimary: {
+      action: 'confirmed',
+    },
+  },
+  parameters: {
+    controls: {
+      include: ['isOpen', 'title', 'secondaryText', 'primaryText'],
+    },
+  },
 };
 
-export const _default = () => {
+export const _default = (args) => {
   return (
-    <Dialog
-      isOpen={boolean('isOpen', true)}
-      onClose={action('closed')}
-      title={text('title', 'my dialog title')}
-      onPrimary={action('confirmed')}
-      secondaryText={text('secondaryText', 'cancel action')}
-      primaryText={text('primaryText', 'confirm action')}
-    >
+    <Dialog {...args}>
```

Comment:
🤔 seems like `onClose` isn't getting passed in. Clicking the close button crashes the story

![image](https://user-images.githubusercontent.com/22185279/151247414-81e7f914-5ff2-4fbe-9a9c-0f4f415285c0.png)

Reply:
Oh yep. I fixed _almost_ all the actions I came across. But I missed a few stories. So this one is still using the old `action` I will be doing another pass to clean up _all_ the actions.